Dockerfile

```diff
@@ -62,36 +73,41 @@ ENV PYENV_ROOT /home/docs/.pyenv
 ENV PATH /home/docs/.pyenv/shims:$PATH:/home/docs/.pyenv/bin
 
 # Install supported Python versions
-RUN pyenv install 2.7.13 && \
-    pyenv install 3.6.2 && \
-    pyenv install 3.5.4 && \
-    pyenv install 3.4.7 && \
-    pyenv install 3.3.6 && \
-    pyenv global 2.7.13 3.6.2 3.5.4 3.4.7 3.3.6
+RUN pyenv install $PYTHON_VERSION_27 && \
+    pyenv install $PYTHON_VERSION_36 && \
+    pyenv install $PYTHON_VERSION_35 && \
+    pyenv install $PYTHON_VERSION_34 && \
+    pyenv install $PYTHON_VERSION_33 && \
+    pyenv global \
+        $PYTHON_VERSION_27 \
+        $PYTHON_VERSION_36 \
+        $PYTHON_VERSION_35 \
+        $PYTHON_VERSION_34 \
+        $PYTHON_VERSION_33
 
 WORKDIR /tmp
 
-RUN pyenv local 2.7.13 && \
+RUN pyenv local $PYTHON_VERSION_27 && \
     pyenv exec pip install -U pip && \
     pyenv exec pip install --only-binary numpy,scipy numpy scipy && \
     pyenv exec pip install pandas matplotlib virtualenv
```

Comment:
Why do we have these packages (and the ones from the previous line) installed by default in our system?

Reply:
We put a few of these on the system library path for virtualenvs that use system libraries

Reply:
I think this made sense when there were some system packages needed to compile them but now that they come as `manylinux` or `whl` packages, besides that we have Conda support, this is probably not necessary anymore from my point of view.